src/app/public/plugin-resources/code-examples/summary-action-bar/tab/summary-action-bar-demo.component.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-summary-action-bar-demo',
+  templateUrl: './summary-action-bar-demo.component.html',
+  styles: [`
+    sky-key-info {
```

Comment:
Again: since this is demo code, is there a comment on why this is necessary or when to use this?

Reply:
See above.